spec/features/user/user_account_flow_spec.rb

```diff
@@ -26,8 +26,9 @@
       fill_in 'user[email]', with: 'eddie@myemail.com'
       fill_in 'user[password]', with: 'superinsecurepassword'
       fill_in 'user[password_confirmation]', with: 'superinsecurepassword'
+      fill_in 'user[telephone_number]', with: '2222222222'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.